apidaze/xmlbuilder/__init__.py

```diff
@@ -0,0 +1,23 @@
+from lxml import etree
+from apidaze.xmlbuilder.nodes.base_node import BaseNode
+
+
+class XMLBuilder():
```

Comment:
I may suggest a name such as `ExternalScriptBuilder` or `ScriptBuilder`.

Reply:
To be more precise, I may suggest a namespace like `apidaze.script` which would export a builder. In that scenario, one may import a builder as follows;

`from apidaze.script import Builder`